### PR TITLE
Shcharbunou: lesson04

### PR DESCRIPTION
**Pull request by ddshcharbunou**
close: #39

### CODE REVIEW

**Comment by ddshcharbunou** [comment]
second commit is mistake

File: shcharbunou/lesson04/maven-webapp-shcharbunouCV/src/main/webapp/style.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+.about-me-photo {
+    width: 40%;
+    float: left;
+}
+
+.about-me-contact {
+    width: 60%;
+    margin-top: 3%;
+    float: left;
+}
```
Comment (rakovets) [inline shcharbunou/lesson04/maven-webapp-shcharbunouCV/src/main/webapp/style.css:72]: "Violation: POSIX standard defines a line: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206
Solution: Add terminating <newline>"

**Review by rakovets — CHANGES_REQUESTED** [review]
You need to return removed files.

**Comment by ddshcharbunou** [comment]
fixed

**Pull request closed without merging**